gtsfm/scene_optimizer.py

```diff
@@ -96,7 +144,8 @@ def create_computation_graph(
         camera_intrinsics_i1_graph: Delayed,
         camera_intrinsics_i2_graph: Delayed,
         exact_intrinsics: bool = True,
-    ) -> Tuple[Delayed, Delayed, Delayed]:
+        i2Ti1_expected_graph: Optional[Delayed] = None,
```

Comment:
i might add 1-sentence to docstring explaining that `i2Ti1_expected_graph` is the ground truth